apps/studio/src/components/tableview/TableTable.vue

```diff
@@ -1325,7 +1453,7 @@ export default Vue.extend({
       pendingUpdate.cell.getElement().classList.remove('edit-error')
     },
     showColumnFilterModal() {
-      this.$modal.show(this.columnFilterModalName)
+      this.$modal.show(this.columnFilterModalName, )
```

Comment:
Since there's no second argument being passed in, take the comma out of there, please and thanks